chart/templates/image-builder-configmap.yaml

```diff
@@ -4,16 +4,6 @@
 {{ $comp := .Values.components.imageBuilder -}}
 {{- $this := dict "root" . "gp" $.Values "comp" $comp -}}
 
-{{- define "registry-name" -}}
```

Comment:
The function is still used below (cmp. [here](https://github.com/gitpod-io/gitpod/pull/4547/files#diff-f8e19dacea608343975006973dd477d4c4e0f27f3f84fbe551b2f9afef1ff917L33-L34)). If we remove here the fallbacks should go away as well.